src/components/CompareResults/SubtestsResults/SubtestsRevisionHeader.tsx

```diff
@@ -0,0 +1,145 @@
+import { type ReactNode } from 'react';
+
+import AppleIcon from '@mui/icons-material/Apple';
+import { Link } from '@mui/material';
+import { style } from 'typestyle';
+
+import { Strings } from '../../../resources/Strings';
+import { Colors, Spacing } from '../../../styles';
+import type {
+  SubtestsRevisionsHeader,
+  PlatformShortName,
+} from '../../../types/state';
+import {
+  getTreeherderURL,
+  truncateHash,
+  getDocsURL,
+} from '../../../utils/helpers';
+import { getPlatformShortName } from '../../../utils/platform';
+import AndroidIcon from '../../Shared/Icons/AndroidIcon';
+import LinuxIcon from '../../Shared/Icons/LinuxIcon';
+import WindowsIcon from '../../Shared/Icons/WindowsIcon';
+
+const styles = {
+  revisionHeader: style({
+    // borderBottom: `0.5px solid ${Colors.BorderDropdownMenu}`,
+    display: 'flex',
+    // justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingBottom: '12px',
+    marginBottom: '12px',
+  }),
+  tagsOptions: style({
+    textAlign: 'right',
+    $nest: {
+      'span:nth-child(3n)': {
+        background: '#592ACB',
+      },
+      'span:nth-child(3n+1)': {
+        background: '#005E5E',
+      },
+      'span:nth-child(3n+2)': {
+        background: '#0250BB',
+      },
+    },
+  }),
+  chip: style({
+    borderRadius: '4px',
+    color: Colors.Background300,
+    // To be removed
+    fontFamily: 'SF Pro',
+    fontStyle: 'normal',
+    fontWeight: 700,
+    fontSize: '8.2px',
+    // End to be removed
+    gap: Spacing.Small + 2,
+    letterSpacing: '0.02em',
+    marginLeft: Spacing.xSmall,
+    padding: Spacing.xSmall,
+    textAlign: 'center',
+    textTransform: 'uppercase',
+  }),
+  typography: style({
+    fontFamily: 'SF Pro',
+    fontStyle: 'normal',
+    fontWeight: 590,
+    fontSize: '15px',
+    lineHeight: '20px',
+  }),
+};
+
+function createTitle(
+  header: SubtestsRevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  if (isLinkSupported) {
+    return (
+      <>
+        <Link
+          aria-label='link to suite documentation'
+          underline='hover'
+          target='_blank'
+          href={docsURL}
+        >
+          {header.suite}
+        </Link>
+      </>
+    );
+  } else {
+    return header.suite;
+  }
+}
+
+function getExtraOptions(extraOptions: string) {
+  return extraOptions ? extraOptions.split(' ') : [];
+}
+
+function SubtestsRevisionHeader(props: SubtestsRevisionHeaderProps) {
+  const { header } = props;
+  const { docsURL, isLinkSupported } = getDocsURL(
+    header.suite,
+    header.framework_id,
+  );
+  const platformIcons: Record<PlatformShortName, ReactNode> = {
```

Comment:
Can be moved to platform.ts -> platform.tsx

```
import { type ReactNode } from 'react';
import AppleIcon from '@mui/icons-material/Apple';
import AndroidIcon from '../components/Shared/Icons/AndroidIcon';
import LinuxIcon from '../components/Shared/Icons/LinuxIcon';
import WindowsIcon from '../components/Shared/Icons/WindowsIcon';


export const platformIcons: Record<PlatformShortName, ReactNode> = {
  Linux: <LinuxIcon />,
  OSX: <AppleIcon />,
  Windows: <WindowsIcon />,
  Android: <AndroidIcon />,
  Unspecified: '',
};
```

Reply:
Honestly, I'd create `platform.tsx` in the `Results` folder. See what others think. I agree it would be confusing to have another `.tsx` file in the utils folder.

Reply:
@julienw What do you think?

Reply:
I added this suggestion in the clean-up [subtask](https://mozilla-hub.atlassian.net/browse/PCF-488)